src/test/java/org/breedinginsight/model/delta/DeltaEntityFactoryUnitTest.java

```diff
@@ -426,7 +426,6 @@ private BrAPIObservation createBrAPIObservation() {
             .additionalInfo(toJsonObject(additionalInfoString))
             .collector("intern")
             .externalReferences(createExternalReferences())
-            .geoCoordinates(BrApiGeoJSON.builder().geometry(null).type("Feature").build())
```

Comment:
What was the reason this was removed?

Reply:
this was a problem I had locally.  It should not have been uploaded.  I will revert it.
-Thanks